validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# pylint disable: inconsistent-return-statements
```

Comment:
Which functions does this apply to? Can it be moved to just those functions, instead of across the entire file?